packages/components/ovh-at-internet/src/ovh-at-internet.ts

```diff
@@ -115,6 +118,10 @@ export default class OvhAtInternet extends OvhAtInternetConfig {
   }
 
   shouldUsePianoAnalytics() {
+    if (window.location?.hostname === 'localhost') {
+      return true;
+    }
```

Comment:
Use environment variable also to enable tracking in localhost.
Environment variable can be defined in `packages/manager/tools/dev-server/bin/manager-dev-server.js`